tests/backup_helper.go

```diff
@@ -2539,22 +2539,28 @@ func ValidateRestore(ctx context1.Context, restoreName string, orgID string, exp
 		// VALIDATION OF VOLUMES
 		log.InfoD("Validating Restored Volumes for the namespace (restoredAppContext) [%s] in restore [%s]", expectedRestoredAppContextNamespace, restoreName)
 
-		// Collect all volumes belonging to a context
+		// Collect all volumes belonging to a context (namespace)
```

Comment:
How about?

```go
Collect all volumes belonging to a namespace
```

Reply:
Modified it, will be a part of next commit.